test_data/test_projects/pipline/preparedata.py

```diff
@@ -0,0 +1,24 @@
+import time
+import sys
+import os
+
```

Comment:
I think the python files in pipline should be refactored to used variables for the expected values, also the directory name should be switched to pipeline.